omero_tagsearch/templates/omero_tagsearch/tagnav.html

```diff
@@ -144,21 +144,28 @@
                 });
 
                 $('#id_operation').change(function() {
-                  $("#tagSearchForm").submit();
+                  if ($('#enableCheckbox').is(':checked'))
+                    $("#tagSearchForm").submit();
```

Comment:
Just a minor comment - You may want to always use `{}` with if statements to avoid possible confusion when adding a 2nd indented statement below this line which would *appear* to be within the `if` block but wouldn't be.

Reply:
you're right, that wouldn't be the first time I get a bug this way. A bad habit that I should change. I'll stop doing that